### PR TITLE
chore: Remove travis, add GHA

### PR DESCRIPTION
**Pull request by attiks**
Refs: #POCAM-69

### CODE REVIEW

File: .github/workflows/docker-build-image.yml [hunk 1]
```diff
@@ -26,9 +26,10 @@ jobs:
         aws_secret_access_key: ${{ secrets.ECR_AWS_ACCESS_KEY_ID }}
         docker_registry_url: public.ecr.aws
         docker_registry_path: /unocha/
-        docker_image: pocam-site
+        docker_image: cerf-site
```
Comment (lazysoundsystem) [inline .github/workflows/docker-build-image.yml]: Are you sure?

**Review by lazysoundsystem — COMMENTED** [review]
The tests aren't building because we still have the preserve-paths package. This should be patched or removed

File: .github/workflows/docker-build-image.yml (hunk 1 above)
Comment (attiks) [reply .github/workflows/docker-build-image.yml]: Nope ;-)

**Comment by attiks** [comment]
Removed

**Comment by lazysoundsystem** [comment]
It's looking for a docker-compose.yml in the tests directory, but not finding one.

**Comment by attiks** [comment]
hmmm

> Successfully tagged unocha/pocam-site:local

> Error response from daemon: manifest for public.ecr.aws/unocha/pocam-site:local not found: manifest unknown: Requested image not found

Credentials?

**Comment by lazysoundsystem** [comment]
Sorry, just seen this. The docker-compose.yml is looking for `public.ecr.aws/unocha/pocam-site:local` but the Makefile is creating `unocha/pocam-site.local` https://github.com/UN-OCHA/pocam8-site/blob/acdaed036a9411562c08b9557f74005ee5a58364/Makefile#L15

File: .github/workflows/run-tests.yml [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+
+      - name: Setup Environment
+        id: docker
+        uses: cafuego/command-output@main
+        with:
+          run: |
+            docker compose -f tests/docker-compose.yml up -d
+            sleep 10
+            docker ps -a
+            docker compose -f tests/docker-compose.yml exec -w /srv/www -T drupal composer -vvv install
```
Comment (lazysoundsystem) [inline .github/workflows/run-tests.yml]: The `-vvv` can be removed - not sure why the tests were failing at this point - nor why they aren't any longer, but the `-vvv` didn't help on that anyway.

File: composer.json [hunk 3]
```diff
@@ -85,34 +84,23 @@
...
             "cweagans/composer-patches": true,
             "dealerdirect/phpcodesniffer-composer-installer": true,
-            "drupal-composer/preserve-paths": true,
             "drupal/core-composer-scaffold": true,
             "drupal/core-project-message": true,
             "drupal/console-extend-plugin": true,
             "oomphinc/composer-installers-extender": true,
-            "symfony/flex": true
+            "symfony/flex": true,
+            "drupal-composer/preserve-paths": true
```
Comment (lazysoundsystem) [inline composer.json:100]: Not sure why composer wanted to add this line, after the package had been removed. We can remove it in a future PR.

File: tests/docker-compose.yml [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+    volumes:
+      - "./settings:/srv/www/shared/settings:ro"
+      # Mount volumes for the private and public files.
+      - "site-public:/srv/www/html/sites/default/files:rw"
+      - "site-private:/srv/www/html/sites/default/private:rw"
+      # Mount the folder for composer scripts.
+      - "../scripts:/srv/www/scripts:ro"
+      # Mount the folders needed for the tests.
+      - "./json:/srv/www/html/sites/default/files/json:rw"
+      - "./yaml:/srv/www/html/sites/default/files/content:rw"
```
Comment (lazysoundsystem) [inline tests/docker-compose.yml]: I don't think we have anything in the json and yaml folders, so they can be removed.

**Review by lazysoundsystem — APPROVED** [review]
Not sure why the tests were failing yesterday, but happy to see them passing now!

File: composer.json (hunk 3 above)
Comment (attiks) [reply composer.json:100]: OK

**Pull request merged**